src/scenic/domains/driving/__init__.py

```diff
@@ -1,5 +1,7 @@
 """Domain for driving scenarios.
 
+This domain must be used with 2D compatibility mode.
```

Comment:
```suggestion
This domain must be used in `2D compatibility mode`.
```

(Likewise for the identical notes in the other files. If the link doesn't connect automatically you might have to add `:ref:` in front, but that shouldn't be necessary.)

Reply:
Actually for the places where we expect to support 3D mode relatively soon, i.e. here and CARLA, let's put "must currently be used in" and add that we expect to add 3D support soon.